bobtask/task.go

```diff
@@ -388,3 +390,41 @@ func keyExists(m map[string]interface{}, key string) bool {
 	_, ok := m[key]
 	return ok
 }
+
+func (t *Task) UnmarshalYAML(value *yaml.Node) (err error) {
+	defer errz.Recover(&err)
+
+	var values struct {
+		Lowercase []string `yaml:"dependson"`
+		Camelcase []string `yaml:"dependsOn"`
+	}
+
+	err = value.Decode(&values)
+	errz.Fatal(err)
+
+	if len(values.Lowercase) > 0 && len(values.Camelcase) > 0 {
+		errz.Fatal(errors.New("both `dependson` and `dependsOn` exists for task"))
```

Comment:
This shouldn't be a user error. Is it possible to annotate the error message with the actual task name?

Reply:
I tried to see if I can add the name, but the key(used for name) is 1-level higher and couldn't manage to get it

Reply:
![line](https://user-images.githubusercontent.com/944919/180444264-8fc4c624-1118-4d22-8147-a0f29d297079.png)

for this value *yaml.Node will have .Line 25. maybe display the line?

Reply:
If this is the best we can get.. Yes, please!